PubSub/src/Subscription.php

```diff
@@ -200,6 +202,15 @@ public function name()
      *           messages, and thus configures how far back in time a `Seek`
      *           can be done. Cannot be more than 7 days or less than 10 minutes.
      *           **Defaults to** 7 days.
+     *     @type array $expirationPolicy A policy that specifies the conditions
+     *           for resource expiration (i.e., automatic resource deletion).
+     *     @type string $expiration.ttl Specifies the "time-to-live" duration
```

Comment:
Could we please notate the fact this expects the string to be formatted as a duration? Also, could we please confirm this is set correctly (especially with the gRPC transport) with a system test?

Reply:
Thanks for catching this, it turned into a little bit of a rabbit hole, but the behavior is corrected in the newest commit.